pathd/path_zebra.c

```diff
@@ -665,6 +723,8 @@ void path_zebra_init(struct event_loop *master)
 	zclient_sync->instance = 1;
 	zclient_sync->privs = &pathd_privs;
 
+	zclient->nexthop_update = path_zebra_nexthop_update;
```

Comment:
I'm not really interested in having a code changes that uses [ZEBRA_NEXTHOP_UPDATE] and then another commit switches this out.  Let's consolidate and shrink this down to 1 commit.

Reply:
Does it look good in the recent push?